operators/Cargo.toml

```diff
@@ -45,3 +45,7 @@ tempfile = "3.1"
 [[bench]]
 name = "workflows"
 harness = false
+
+[[bench]]
+name = "pip"
```

Comment:
point in polygon? Bzw. ist das kein workflow?

Reply:
it is a workflow but the other bench suite doesn't work so I created a new on